alg/gdaltransformer.cpp

```diff
@@ -2157,6 +2165,9 @@ const char *GDALGetGenImgProjTranformerOptionList(void)
  * SRC_GEOLOC_ARRAY description for details, assumptions, and defaults. If this
  * option is set, DST_METHOD=GEOLOC_ARRAY will be assumed if not set.
  * </li>
+ * <li>GEOLOC_NORMALIZE_LONGITUDE_MINUS_180_PLUS_180=YES/NO. (GDAL &gt;= 3.12.0)
+ * Whether to force geolocation longitudes into -180,180 when longitude/latitude is
+ * the coordinate system of the geolocation arrays. The default is NO.
```

Comment:
actually the default is to enable this mode when the values in the geolocation array are in the -180,180 range (cf gdalgeoloc.cpp line 200)